pipelines/tasks/utils/gcsutils.py

```diff
@@ -0,0 +1,67 @@
+"""
+Utility functions that interact with Google Cloud Storage.
+For instance, copying files from Databricks volumes into GCP buckets.
+"""
+
+from google.cloud import storage
+from google.cloud.storage.bucket import Bucket
+from databricks.sdk.runtime import dbutils  # noqa: F401
+import os
+
+
+def save_file(
+    bucket: Bucket, src_volume_filepath: str, dest_bucket_pathname: str
+) -> None:
+    """Save file from databricks volume to a GCP bucket path."""
+    blob = bucket.blob(dest_bucket_pathname)
+    if blob.exists():
+        raise ValueError(dest_bucket_pathname + ": File already exists in bucket.")
+    blob.upload_from_filename(src_volume_filepath)
+
+
+def publish_inference_output_files(
+    db_workspace: str,
+    institution_name: str,
+    external_bucket_name: str,
+    sst_job_id: str,
+    approved: bool,
+) -> str:
+    """
+    Publish output files to bucket, with folder determined based on approved parameter,
+    and return the status of the job as a string.
+    """
+    # Destination: The output directory in the bucket that the files get moved to.
+    bucket_directory = f"{'approved' if approved else 'unapproved'}/{sst_job_id}"
```

Comment:
nit: i think this is technically correct, but python has a cleaner syntax for conditional assignment:

```suggestion
    bucket_directory = "approved" if approved else f"unapproved/{sst_job_id}"
```

Reply:
oops, sure enough! i misread your code. imo that underscores my point: doing this in an f-string is straight-up harder to read, even though it's technically allowed :) too late now, and no big deal